lib/travis/build/templates/header.sh

```diff
@@ -28,6 +28,24 @@ export SHELL
 export TERM
 export USER
 
+case $(uname | tr '[A-Z]' '[a-z]') in
+  linux)
+    export TRAVIS_OS_NAME=linux
+    ;;
+  *)
+    export TRAVIS_OS_NAME=osx
+    ;;
+esac
+
+if [[ "$TRAVIS_OS_NAME" = linux ]]; then
```

Comment:
I'm pretty sure `shellcheck` prefers `[[` to be used with `==` equality checks, FYI!